docs/INSTALL-DOCKER-COMPOSE.md

```diff
@@ -138,12 +187,12 @@ Also, a superuser account for the Open Klant UI on http://localhost:8002 is crea
 
 ## Stopping
 
-1. Stop ZAC.
-2. Stop the Docker containers by executing the command: `docker compose down`.
+1. Stop ZAC (only if you are running ZAC separately and not as part of the Docker Compose setup)
+2. Stop all Docker containers by executing the command: `./stop-docker-compose.sh`.
```

Comment:
This file doe snot exist

Reply:
Yes it does? It's in the root folder, next to the start up scripts.